configure.ac

```diff
@@ -2961,8 +2961,8 @@ AS_VAR_IF([have_uuid], [missing], [
       AC_DEFINE([HAVE_UUID_H], [1])
       AC_DEFINE([HAVE_UUID_GENERATE_TIME_SAFE], [1])
     ], [
-      LIBUUID_LIBS="-luuid"
-      LIBUUID_CFLAGS=
+      LIBUUID_CFLAGS=${LIBUUID_CFLAGS:-""}
+      LIBUUID_LIBS=${LIBUUID_LIBS:-"-luuid"}
```

Comment:
This messes up the macOS build. How about something like this? Side effect: It would prefer libc versions of `uuid_generate_time` to `-luuid`.

```diff
diff --git a/configure.ac b/configure.ac
index 17a9aea02e..c1d86c9c07 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2961,11 +2961,10 @@ AS_VAR_IF([have_uuid], [missing], [
       AC_DEFINE([HAVE_UUID_H], [1])
       AC_DEFINE([HAVE_UUID_GENERATE_TIME_SAFE], [1])
     ], [
-      LIBUUID_CFLAGS=${LIBUUID_CFLAGS:-""}
-      LIBUUID_LIBS=${LIBUUID_LIBS:-"-luuid"}
       AC_CHECK_HEADERS([uuid/uuid.h], [
-        WITH_SAVE_ENV(
-          [AC_CHECK_LIB([uuid], [uuid_generate_time], [have_uuid=yes])
+        WITH_SAVE_ENV([
+          AC_SEARCH_LIBS([uuid_generate_time], [uuid], [have_uuid=yes])
+          LIBUUID_LIBS=${LIBUUID_LIBS:-$LIBS}
         ])
         WITH_SAVE_ENV([
           AC_CHECK_LIB([uuid], [uuid_generate_time_safe], [
@@ -2974,21 +2973,12 @@ AS_VAR_IF([have_uuid], [missing], [
           ])
         ])
       ])
+      LIBUUID_CFLAGS=${LIBUUID_CFLAGS:-""}
+      LIBUUID_LIBS=${LIBUUID_LIBS:-"-luuid"}
     ]
   )
 ])
 
-dnl macOS has uuid/uuid.h but uuid_generate_time is in libc
-AS_VAR_IF([have_uuid], [missing], [
-  AC_CHECK_HEADERS([uuid/uuid.h], [
-    AC_CHECK_FUNC([uuid_generate_time], [
-      have_uuid=yes
-      LIBUUID_CFLAGS=${LIBUUID_CFLAGS:-""}
-      LIBUUID_LIBS=${LIBUUID_LIBS:-""}
-    ])
-  ])
-])
-
 AS_VAR_IF([have_uuid], [missing], [have_uuid=no])
 
 # 'Real Time' functions on Solaris
```

Reply:
I used a slightly different approach:
```
        AS_VAR_IF([have_uuid], [yes], [
          LIBUUID_CFLAGS=${LIBUUID_CFLAGS-""}
          LIBUUID_LIBS=${LIBUUID_LIBS-"-luuid"}
        ])
```

Reply:
Nice, that works swell!